.gitignore

```diff
@@ -1,3 +1,4 @@
 book
+mdbook
```

Comment:
Why is `mdbook/` ignored while there is a file `mdbook/download.cmd` committed?

Reply:
Ok. But, could you rename the folder name to something less confusing than `mdbook/`? E.g. `bin\`.
I think it might confuse people when they see `mdbook\mdbook.exe`.